aggregations-and-analytics/12_lag/12_lag.md

```diff
@@ -0,0 +1,43 @@
+# 12 Lag
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.13%2B-lightgrey)
+
+> :bulb: This example will show how to retrieve the previous value and compute trends for a specific data partition.
+
+The source table (`fake_stocks`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates fake stock quotation in memory based on Java Faker expressions.
+
+In this recipe we're using the [LAG](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/functions/systemfunctions/#aggregate-functions) function to retrieve the previous value for a specific stock and then calculate its trend.
```

Comment:
```suggestion
In this recipe we're using the [LAG](https://nightlies.apache.org/flink/flink-docs-stable/docs/dev/table/functions/systemfunctions/#aggregate-functions) function to retrieve the previous row value for a specific stock and then calculate its trend.
```

Reply:
Perhaps it's good to add a little more detail on the use case: that we're going to create a table which contains stock ticker updates and we want to determine if the new stock price has gone up or down compared to its previous stock price. First we create the table and then you use a combination of a select statement and a case to determine  what has happened with the stockprice? WDYT?

Reply:
I agree! Reworded the example to give a bit more explanation